api/swagger/server/restapi/certHandler.go

```diff
@@ -0,0 +1,226 @@
+package restapi
+
```

Comment:
That was my first purpose , it would be cleaner but in mkcert none of the functions are public :/
And we were thinking it's okay to take this because mkcert is not gonna to be a product super updated in time

Reply:
Maybe i was not clear.

Mkcert is a tool with unique goal. And you can't pick what you want on it. You call Run and thats it. And i didn't need all of the tool and my using was not aligned with the goal of mkcert